scriptmodules/supplementary/sdl2.sh

```diff
@@ -88,7 +88,7 @@ function build_sdl2() {
     sed -i 's/libgl1-mesa-dev,/libgl1-mesa-dev, '"${conf_depends[*]}"'/' ./debian/control
     sed -i 's/confflags =/confflags = '"${conf_flags[*]}"' \\\n/' ./debian/rules
 
-    if isPlatform "rpi"; then
+    if isPlatform "rpi" && [[ -d "/opt/vc/include" ]]; then
```

Comment:
Rather than applying a workaround for the header issue, perhaps it would be better/cleaner to stop building SDL2 with both drivers, and just remove these blocks entirely? You won't need to change anything in the SDL source, but on the next branch update/rebase, you can leave out this commit: https://github.com/RetroPie/SDL-mirror/commit/2db72c15114a190d18ebac343aafd3f53c4d849a

Just a suggestion, as I recall @hhromic recently mentioned some kind of issue with some SDL2 software not working correctly unless the KMS driver was forced via environment variable (IIRC).

Reply:
The latest source of SDL2 can be built with x11, egl, and kms/drm support simultaneously for a Pi4. If RA is also built with the same support (x11, egl, kms/drm) support as well, the user can use both the desktop environment and headless without any performance impact.